File: combinationlock.py
cnt = 0
n = int(input())
s1 = input()
s2 = input()
for i in range(n):
    cnt += min(abs(int(s1[i]) - int(s2[i])), (10 - abs(int(s1[i]) - int(s2[i]))))
print(cnt)

File: combinationlock.java
import java.util.Scanner;

public class combinationlock {

	public static void main(String[] args) {

		Scanner in = new Scanner(System.in);

		int cnt = 0;
		int n = in.nextInt();

		String s1 = in.next();
		String s2 = in.next();

		for (int i = 0; i < n; i++) {

			int n1 = Character.getNumericValue(s1.charAt(i));
			int n2 = Character.getNumericValue(s2.charAt(i));

			cnt += Math.min(Math.abs(n1 - n2), 10 - Math.abs(n1 - n2));

		}

		System.out.println(cnt);

	}
}
